src/components/select-field/__tests__/BaseSelectField.test.js

```diff
@@ -1000,25 +1022,60 @@ describe('components/select-field/BaseSelectField', () => {
 
     describe('openDropdown()', () => {
         test('should set isOpen state to true when called', () => {
+            const event = {};
             const wrapper = shallowRenderSelectField();
             const instance = wrapper.instance();
 
-            instance.openDropdown();
+            instance.openDropdown(event);
 
             expect(wrapper.state('isOpen')).toBe(true);
         });
 
+        test.each([
+            [
+                { target: document.createElement('div') },
+                {
+                    top: 'unset',
+                    left: 'unset',
+                    minWidth: 'unset',
+                    transform: 'none',
+                    willTransform: 'unset',
+                    width: 120,
+                },
+            ],
+            [{}, {}],
+        ])('%o should set dropdownStyle state when position is fixed', (event, expected) => {
+            Element.prototype.getBoundingClientRect = jest.fn(() => {
```

Comment:
we probably should not do this; this is modifying *all* instances of element's definition of `getBoundingClientRect`. There may be a way to use `mockReturnValueOnce` here